### PR TITLE
New package: Microsoft.UI.Xaml.2.7 version 2.7.3

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

* Related to #111840

### CODE REVIEW

File: manifests/m/Microsoft/UI/Xaml/2/7/2.7.3/Microsoft.UI.Xaml.2.7.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.UI.Xaml.2.7
+PackageVersion: 2.7.3
+InstallerType: appx
+UpgradeBehavior: install
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/UI/Xaml/2/7/2.7.3/Microsoft.UI.Xaml.2.7.installer.yaml:7]:
```suggestion
UpgradeBehavior: install
MinimumOSVersion: 10.0.15063.0
Platform:
  - Windows.Universal
AppsAndFeaturesEntries:
  - DisplayVersion: 7.2208.15002.0
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
Looks like the validation pipeline is too strict on Appx  packages, saying that Winget Manifest PackageVersion must match the version in the Appx Manifest, even though Microsoft.UI.Xaml uses the marketing version and I've specified the real version in AppsAndFeaturesEntries

**Comment by mdanish-kh** [comment]
Seems to be a similar case to Microsoft.PowerShell.Preview where we would need a manual merge if we want PackageVersion 2.7.3. The DisplayVersion retrieved from the MSIX is `7.2208.15002.0` and pipelines don't like it when PackageVersion is different from MSIX DisplayVersion

[Policy] Needs-Manual-Merge

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**